priv/templates/phoenix.gen.model/model.ex

```diff
@@ -3,7 +3,7 @@ defmodule <%= module %> do
 
   schema <%= inspect plural %> do
 <%= for {k, _} <- attrs do %>    field <%= inspect k %>, <%= inspect types[k] %><%= schema_defaults[k] %>
-<% end %><%= for {k, _, m, _} <- assocs do %>    belongs_to <%= inspect k %>, <%= m %>
+<% end %><%= for {k, k_id, m, _} <- assocs do %>    belongs_to <%= inspect k %>, <%= m %><%= if !String.ends_with?(Atom.to_string(k_id), "_id") do %>, define_field: false, foreign_key: <%= inspect k %><% end %>
```

Comment:
Will this work as well? If we don't define the field... then we won't be able to use it in queries.